python/dronin-logfsimport

```diff
@@ -12,10 +12,10 @@ def main():
 
```

Comment:
Should update the help on the line above.

(why does Github restrict which lines you can comment on now? 👎 )